pytorch_lightning/testing/no_val_end_module.py

```diff
@@ -80,14 +80,14 @@ def loss(self, labels, logits):
         nll = F.nll_loss(logits, labels)
         return nll
 
-    def training_step(self, data_batch, batch_i):
+    def training_step(self, batch, batch_i):
         """
         Lightning calls this inside the training loop
-        :param data_batch:
+        :param batch:
```

Comment:
add variable type and description like 
```
:param tuple(ndarray,ndarray) batch: pair of data and desired labels...
```

Reply:
May as well use the typing module since this project is 3.6+ anyway, and it's natively supported, plus adding that improves things like autocomplete.

Reply:
Also, I think that's a separate PR, since this is about renaming, not adding type info.